packages/examples/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@api3/examples",
+  "version": "0.1.0",
+  "license": "MIT",
+  "description": "A collection of examples showcasing the features of Airnode",
+  "scripts": {
+    "build": "hardhat compile",
+    "choose-integration": "ts-node scripts/choose-integration.ts",
+    "create-airnode-secrets": "ts-node scripts/create-airnode-secrets.ts",
+    "create-aws-secrets": "ts-node scripts/create-aws-secrets.ts",
+    "deploy-airnode": "ts-node scripts/deploy-airnode.ts",
+    "deploy-requester": "ts-node scripts/deploy-requester.ts",
+    "deploy-rrp": "ts-node scripts/deploy-rrp.ts",
+    "derive-and-fund-sponsor-wallet": "ts-node scripts/derive-and-fund-sponsor-wallet.ts",
+    "eth-node": "hardhat node",
+    "make-request": "ts-node scripts/make-request.ts",
+    "print-account-details": "ts-node scripts/print-account-details.ts",
+    "rebuild-airnode-container": "ts-node scripts/rebuild-airnode-container.ts",
+    "rebuild-artifacts-container": "ts-node scripts/rebuild-artifacts-container.ts",
+    "rebuild-deployer-container": "ts-node scripts/rebuild-deployer-container.ts",
+    "remove-airnode": "ts-node scripts/remove-airnode.ts",
+    "run-airnode-locally": "ts-node scripts/run-airnode-locally.ts",
+    "sponsor-requester": "ts-node scripts/sponsor-requester.ts"
+  },
+  "dependencies": {
+    "@api3/admin": "^0.1.0",
+    "@api3/airnode-abi": "^0.1.0",
+    "@api3/deployer": "^0.1.0",
+    "@api3/protocol": "^0.1.0",
```

Comment:
Just tried from scratch again and I had to add `"@types/prompts": "^2.0.14",` under `dependencies` (rather than it being just in `devDependencies`), otherwise I was getting the following on `yarn choose-integration`:
```bash
$ ts-node scripts/choose-integration.ts
/home/ec2-user/airnode/node_modules/ts-node/src/index.ts:692
    return new TSError(diagnosticText, diagnosticCodes);
           ^
TSError: ⨯ Unable to compile TypeScript:
scripts/choose-integration.ts:3:39 - error TS7016: Could not find a declaration file for module 'prompts'. '/home/ec2-user/airnode/node_modules/prompts/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/prompts` if it exists or add a new declaration (.d.ts) file containing `declare module 'prompts';`

3 import prompts, { PromptObject } from 'prompts';
```

Otherwise, I was able to get through the whole local flow and make multiple requests.

Reply:
That should not be needed. I guess you run `yarn install` from a different branch and that removed these packages from node modules.

If you just run `yarn install` from this branch (without moving it to dependencies) it will work.